guides/common/modules/proc_using-remote-execution-templates-on-flatpak.adoc

```diff
@@ -0,0 +1,47 @@
+[id="using-remote-execution-templates-on-flatpak_{context}"]
```

Comment:
I don't think you need this module. Running remote execution jobs is described in the Managing hosts guide. I don't think that adding new templates justifies three new procedures that end up duplicating REX documentation in Managing hosts.

Reply:
Or maybe (if this is a new remote job category called Flatpak -- I don't know) you could add this to https://docs.theforeman.org/nightly/Managing_Hosts/index-katello.html#default-job-template-categories_managing-hosts

Reply:
I've got a question out to @sjha4 about this.

Reply:
>> if this is a new remote job category called Flatpak -- I don't know)
It's under katello, we don't have a new category for flatpak..It'd be good to keep the templates in the flatpak section but I am fine keeping things consistent.